### PR TITLE
added recipe for drivelib, a python API for Google Drive

### PR DESCRIPTION
**Pull request by notestaff**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by notestaff** [comment]
@conda-forge/staged-recipes

File: recipes/drivelib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8e00c9981225ed7eae9f217f12242d98130aff59c2ff3e6026570589968a8fe3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # py<37
```
Comment (marcelotrevisani) [inline recipes/drivelib/meta.yaml:15]:
```suggestion
```

File: recipes/drivelib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
+    - google-api-python-client
+    - google-auth
+    - google-auth-httplib2
+    - google-auth-oauthlib
+    - oauth2client
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/drivelib/meta.yaml:25]: I think you just need ``pip`` and ``python`` here
```suggestion
    - pip
    - python >=3.7
```

File: recipes/drivelib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  run:
+    - google-api-python-client
+    - google-auth
+    - google-auth-httplib2
+    - google-auth-oauthlib
+    - oauth2client
+    - python
```
Comment (marcelotrevisani) [inline recipes/drivelib/meta.yaml:32]:
```suggestion
    - python >=3.7
```

**Comment by notestaff** [comment]
p.s. this library is needed to upgrade https://github.com/conda-forge/git-annex-remote-googledrive-feedstock to the  current version.

File: recipes/drivelib/meta.yaml (hunk 3 above)
Comment (notestaff) [reply recipes/drivelib/meta.yaml:27]: @marcelotrevisani wasn't this the cause of the linter complaint above, https://github.com/conda-forge/staged-recipes/pull/11786#issuecomment-636649864

File: recipes/drivelib/meta.yaml (hunk 1 above)
Comment (notestaff) [reply recipes/drivelib/meta.yaml:15]: @marcelotrevisani when I tried to do this instead by requiring 'python >=3.7' I get a linter warning.  Should I still do it that way?

Comment (marcelotrevisani) [reply recipes/drivelib/meta.yaml:15]: You can add this package as ``noarch: python`` and let the python constrains as I suggested

Comment (marcelotrevisani) [reply recipes/drivelib/meta.yaml:15]: Please see here for more information
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

Comment (marcelotrevisani) [reply recipes/drivelib/meta.yaml:15]: Here you can find one example of recipe which uses the same constrains and it is ``noarch: python``
https://github.com/conda-forge/grayskull-feedstock/blob/master/recipe/meta.yaml

**Comment by notestaff** [comment]
@marcelotrevisani thx for the pointers, now changed to noarch

**Pull request merged**